### PR TITLE
docs: active channel description

### PR DESCRIPTION
**Pull request by seantking**
## Description

closes: #704

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

**Comment by seantking** [comment]
@damiannolan @crodriguezvega @colin-axner please nit away. Not sure if we want this to have it's own section or be a part of another etc. Open to ideas :+1:  thanks

**Comment by seantking** [comment]
CI failed for a dead link that is actually alive (also not related to this PR) :shrug:

File: docs/app-modules/interchain-accounts/active-channels.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<!--
+order: 3
+-->
+
+# Understanding Active Channels 
+
+The Interchain Accounts module uses [ORDERED channels](https://github.com/cosmos/ibc/tree/master/spec/core/ics-004-channel-and-packet-semantics#ordering) to maintain the order of transactions when sending packets from a controller to a host chain. A limitation when using ORDERED channels is that when a packet times out the channel will be closed. 
+
+In the case of a channel closing a Controller chain needs to be able to regain access to the interchain account registered on this channel. `Active Channels` enable this functionality. Future versions of the ICS-27 protocol and the Interchain Accounts module will likely use a new
```
Comment (crodriguezvega) [inline docs/app-modules/interchain-accounts/active-channels.md:9]:
```suggestion
In the case of a channel closing, a controller chain needs to be able to regain access to the interchain account registered on this channel. `Active Channels` enable this functionality. Future versions of the ICS-27 protocol and the Interchain Accounts module will likely use a new 
```

File: docs/app-modules/interchain-accounts/active-channels.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+-->
+
+# Understanding Active Channels 
+
+The Interchain Accounts module uses [ORDERED channels](https://github.com/cosmos/ibc/tree/master/spec/core/ics-004-channel-and-packet-semantics#ordering) to maintain the order of transactions when sending packets from a controller to a host chain. A limitation when using ORDERED channels is that when a packet times out the channel will be closed. 
+
+In the case of a channel closing a Controller chain needs to be able to regain access to the interchain account registered on this channel. `Active Channels` enable this functionality. Future versions of the ICS-27 protocol and the Interchain Accounts module will likely use a new 
+channel type that provides ordering of packets without the channel closing on timing out, thus removing the need for `Active Channels` entirely.  
+
+When an Interchain Account is registered using the `RegisterInterchainAccount` API, a new channel is created on a particular port. During the `OnChanOpenAck` and `OnChanOpenConfirm` steps (Controller & Host chain) the `Active Channel` for this interchain account
```
Comment (crodriguezvega) [inline docs/app-modules/interchain-accounts/active-channels.md:12]: I think we don't use uppercase for controller and host in the other sections of the docs...

```suggestion
When an Interchain Account is registered using the `RegisterInterchainAccount` API, a new channel is created on a particular port. During the `OnChanOpenAck` and `OnChanOpenConfirm` steps (controller & host chain) the `Active Channel` for this interchain account
```

File: docs/app-modules/interchain-accounts/active-channels.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+is stored in state.
+
+It is possible to create a new channel using the same controller chain portID if the previosly set `Active Channel` is now in a `CLOSED` state. This channel creation can be initialized programatically by sending a new `OnChanOpenInit` message like so:
```
Comment (crodriguezvega) [inline docs/app-modules/interchain-accounts/active-channels.md:15]:
```suggestion
It is possible to create a new channel using the same controller chain portID if the previously set `Active Channel` is now in a `CLOSED` state. This channel creation can be initialized programatically by sending a new `OnChanOpenInit` message like so:
```

File: docs/app-modules/interchain-accounts/active-channels.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+```go
+	msg := channeltypes.NewMsgChannelOpenInit(portID, string(versionBytes), channeltypes.ORDERED, []string{connectionID}, icatypes.PortID, icatypes.ModuleName)
+	handler := k.msgRouter.Handler(msg)
+```
+
+Alternatively, any relayer operator may initiate a new channel handshake for this interchain account once the previously set `active channel` is in a `CLOSED` state. This is done by initiating the channel handshake on the controller chain using the same portID associated with the interchain account in question.
```
Comment (crodriguezvega) [inline docs/app-modules/interchain-accounts/active-channels.md:22]:
```suggestion
Alternatively, any relayer operator may initiate a new channel handshake for this interchain account once the previously set `Active Channel` is in a `CLOSED` state. This is done by initiating the channel handshake on the controller chain using the same portID associated with the interchain account in question.  
```

**Review by crodriguezvega — DISMISSED** [review]

**Review by colin-axner — APPROVED** [review]
LGTM, thanks!

**Review by damiannolan — APPROVED** [review]

**Pull request merged**